user/TASK/shoot_task/shoot_task.c

```diff
@@ -209,16 +212,29 @@ static void shoot_reversed_control(Shoot_Motor_t *trigger_motor, Shoot_Motor_t *
     trigger_motor->speed_set = TRIGGER_SPEED * DIR_REVERSED;
 }
 
-
 /**
  * @brief Trigger motor and hopper motor continuously spinning to allow for rapid firing
  * @param Trigger motor and hopper motor structs
  * @retval None
  */
 static void shoot_rapid_control(Shoot_Motor_t *trigger_motor, Shoot_Motor_t *hopper_motor)
 {
-    //Hopper and trigger motor both spinning, used for clearing bullets
-    hopper_motor->speed_set = HOPPER_SPEED;
+    shoot_count++;
+    
+    if(shoot_count > SHOOT_COUNT_MAX){
+        shoot_count = 0;
+        hopper_motor->speed_set = HOPPER_SPEED;
+
+    }
+    if(shoot_count > SHOOT_COUNT_SWTICH){ 
```

Comment:
Put a newline here to make it clear that these are two different ifs

Reply:
Hey we actually need codemaid lol